src/core/server/saved_objects/saved_objects_service.ts

```diff
@@ -223,11 +234,12 @@ export class SavedObjectsService
 
     const {
       savedObjectSchemas: savedObjectsSchemasDefinition,
-      savedObjectMappings,
+      savedObjectMappings: legacyMappings,
       savedObjectMigrations,
       savedObjectValidations,
     } = setupDeps.legacyPlugins.uiExports;
-    this.mappings = savedObjectMappings;
+
+    this.mappings = convertLegacyMappings(legacyMappings || []);
```

Comment:
Even if it should never be `undefined`, there were some integration tests starting the kibana server where the mappings were undefined. As the `uiExport` gathering/reducing code is an unreadable thing, doing this `|| []` here was the pragmatic solution (and will be deleted post 8.0 anyway)